debugger.c

```diff
@@ -12,7 +12,22 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <inttypes.h>
+
+#if __APPLE__
+#include <TargetConditionals.h>
+#if TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE
+#include "SDL.h"
```

Comment:
if you add some magic path to your includes in Xcode, you shouldn't have to special case `#include "SDL.h"` over `#include <SDL.h>`

Reply:
👍